City-of-Pori/pori-kada
# PORi-376 german and swedish language configuration

ATTENTION! Front pages need to be translated and then the translations need to be pointed out as each language versions front page on production.

To test:

1. Sync from production to get all the menus
2. drush fra -y && drush cc all
3. Make sure Swedish is enabled.
3. Go to add menu links to all menus so that they get rendered. There is ton of those. Takes a while.
4. You need to comment the function pori_visitpori_override_feature_language_switch_links_alter to see all the language switcher links.
5. Go to different languages on local.pori.fi and local.visitpori.fi and make sure there is all the menus visible.
6. Translate front pages and change correct nodes to point as language version's front page.
7. Translate attraction card on visitpori and make sure that the everything works.
